src/Core/Prefab/Globe/GlobeLayer.js

```diff
@@ -37,27 +47,48 @@ class GlobeLayer extends TiledGeometryLayer {
      */
     constructor(id, object3d, config = {}) {
         // Configure tiles
-        const schemeTile = globeSchemeTileWMTS(globeSchemeTile1);
+        const schemeTile = [
+            new Extent('EPSG:4326', -180, 0, -90, 90),
+            new Extent('EPSG:4326', 0, 180, -90, 90)];
         const builder = new BuilderEllipsoidTile();
+
         super(id, object3d || new THREE.Group(), schemeTile, builder, config);
 
         this.options.defaultPickingRadius = 5;
+        this.sseSubdivisionThreshold = this.sseSubdivisionThreshold || 1.0;
 
         this.extent = this.schemeTile[0].clone();
 
         for (let i = 1; i < this.schemeTile.length; i++) {
             this.extent.union(this.schemeTile[i]);
         }
-
-        this.culling = globeCulling(2);
-        this.subdivision = globeSubdivisionControl(2,
-            config.maxSubdivisionLevel || 18,
-            config.sseSubdivisionThreshold || 1.0,
-            config.maxDeltaElevationLevel || 4);
     }
 
     preUpdate(context, changeSources) {
-        preGlobeUpdate(context, this);
+        // We're going to use the method described here:
+        //    https://cesiumjs.org/2013/04/25/Horizon-culling/
+        // This method assumes that the globe is a unit sphere at 0,0,0 so
+        // we setup a world-to-scaled-ellipsoid matrix4
+        worldToScaledEllipsoid.getInverse(this.object3d.matrixWorld);
+        worldToScaledEllipsoid.premultiply(
+            new THREE.Matrix4().makeScale(
+                1 / ellipsoidSizes().x,
+                1 / ellipsoidSizes().y,
+                1 / ellipsoidSizes().z));
+
+        // pre-horizon culling
+        cameraPosition.copy(context.camera.camera3D.position).applyMatrix4(worldToScaledEllipsoid);
+
+        // pre-sse
+        const canvasSize = context.view.mainLoop.gfxEngine.getWindowSize();
+        const hypotenuse = canvasSize.length();
+        const radAngle = context.view.camera.camera3D.fov * Math.PI / 180;
+
+         // TODO: not correct -> see new preSSE
+        // const HFOV = 2.0 * Math.atan(Math.tan(radAngle * 0.5) / context.camera.ratio);
+        const HYFOV = 2.0 * Math.atan(Math.tan(radAngle * 0.5) * hypotenuse / canvasSize.x);
+
+        context.camera.preSSE = hypotenuse * (2.0 * Math.tan(HYFOV * 0.5));
```

Comment:
the preSSE is wrong
The correct formula is 
```js
const height = context.camera.height;
const FOV = THREE.Math.degToRad(context.camera.camera3D.fov);
const preSSE = height / (2.0 * Math.tan(FOV * 0.5));
```
we should think about fix it and testing the performance

Reply:
I changed the SSE in the last commit, I'll let you take a look to how its changing, but we can fix it in another PR if you want.